data-model/fdo-types/0.3.0/shared-models/schema/assertion.json

```diff
@@ -0,0 +1,108 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/fdo-types/0.3.0/shared-models/assertions.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "$comment": "Annotation Version 0.3.0",
+  "properties": {
+    "@id": {
+      "type": "string",
+      "description": "The identifier for the digital object."
+    },
+    "@type": {
+      "type": "string",
+      "const": "ods:Assertion",
+      "description": "The type of the digital object, in this case a ods:Assertion"
```

Comment:
yes this would unfortunately be helpful elsewhere too...

Reply:
missing for `media/digital-entity` but i'm not sure if you want to address that this PR